static/app/utils/theme/index.tsx

```diff
@@ -1,5 +1,10 @@
 import CommonTheme from './theme';
 
+/**
+ * Do not import theme values directly as they only define light color theme.
+ * Consuming it directly means that you won't get the correct colors in dark mode.
+ * @deprecated use useTheme hook instead.
```

Comment:
For the actual type don't we want people using the emotion `Theme` type?

Reply:
Good call, let me deprecate that one too